File: app/src/main/java/com/BlizzardArmory/warcraft/equipment/EnchantmentSlot.java
package com.BlizzardArmory.warcraft.equipment;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * The type Enchantment slot.
 */
public class EnchantmentSlot {

    @SerializedName("id")
    @Expose
    private int id;
    @SerializedName("type")
    @Expose
    private String type;

    /**
     * Gets id.
     *
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * Sets id.
     *
     * @param id the id
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * Gets type.
     *
     * @return the type
     */
    public String getType() {
        return type;
    }

    /**
     * Sets type.
     *
     * @param type the type
     */
    public void setType(String type) {
        this.type = type;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/equipment/EnchantmentSlot.kt
package com.BlizzardArmory.model.warcraft.equipment

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Enchantment slot.
 */
@Keep
data class EnchantmentSlot(

    @SerializedName("id")
    var id: Long,

    @SerializedName("type")
    var type: String

)